env.yml

```diff
@@ -41,7 +41,7 @@ dependencies:
   - pytorch_scatter >=2.0
 
   # chemistry
-  - rdkit
+  - rdkit == 2024.03.4
```

Comment:
Why?? And why not `<= 2024.03.3`?

Reply:
The latest version of `rdkit` was not being installed with the required files that are imported by `graphium_cpp`. So, I pinned it to a very recent version. We could do `<= 2024.03.4`, but can we confirm that all earlier versions of `rdkit` contain the required files needed by `graphium_cpp`?

Reply:
Let me also clean my `conda` cache and confirm that it wasn't just a local cache issue with the latest version of `rdkit`

Reply:
Confirmed that the latest version of `rdkit` on `conda` does not include the required files for `graphium_cpp`

Reply:
Yeah this was something that @ndickson-nvidia mentioned as well.

I will merge the PR